src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_postgres.py

```diff
@@ -261,8 +264,12 @@ def _flexible_parameter_update(client, server_name, configuration_name, resource
     return client.update(resource_group_name, server_name, configuration_name, value, source)
 
 
-def _flexible_list_skus(client, location):
-    return client.execute(location)
+def _flexible_list_skus(cmd, client, location, json=None):
+    if not json or json=='false':
+        set_output_format(cmd.cli_ctx, 'table')
+    result = client.execute(location)
+    logger.warning('For prices please refer to https://azure.microsoft.com/en-us/pricing/details/postgresql/server/')
```

Comment:
This has to be a short url. Check in the code for aka.ms/ postgres and mysql link.

Reply:
Added the links!